lightning/src/ln/offers_tests.rs

```diff
@@ -1332,6 +1332,16 @@ fn fails_sending_invoice_without_blinded_payment_paths_for_offer() {
 
 	let invoice_error = extract_invoice_error(david, &onion_message);
 	assert_eq!(invoice_error, InvoiceError::from(Bolt12SemanticError::MissingPaths));
+
+	// Confirm that david drops this failed payment from his pending outbound payments.
+	let events = david.node.get_and_clear_pending_events();
+	assert_eq!(events.len(), 1);
+	match events[0] {
+		Event::InvoiceRequestFailed { payment_id: pay_id } => {
+			assert_eq!(pay_id, payment_id)
+		},
+		_ => panic!("Unexpected Event"),
+	}
```

Comment:
Let's use `get_event!` as seen elsewhere in this file.

Reply:
Done! [58f8f6a](https://github.com/lightningdevkit/rust-lightning/commit/58f8f6a8f5d2ac1ea07d45feab933784fa98bb09)
Thanks for the suggestion!